src/gui.c

```diff
@@ -13910,6 +13911,7 @@ static void about_dialog(GtkMenuItem *menu_item, gpointer data)
 "This program is written by Keldon Jones, and the source code is licensed \
 under the GNU General Public License.\n\n\
 The interface enhancement release " RELEASE " is written by B. Nordli.\n\n\
```

Comment:
This line is no longer correct. I suggest "Interface enhancements added by B. Nordli."